entrypoint.sh

```diff
@@ -18,5 +18,8 @@ fi
 
 sudo install-runner
 
-./config.sh --unattended --replace --url "${RUNNER_URL}" --token "${RUNNER_TOKEN}"
+# Reconfigure from the clean state in case of runner failures/restarts
+./config.sh remove --token "${RUNNER_TOKEN}"
```

Comment:
does this succeed if it's not registered?

Reply:
Hey David. Yes it does.
```
# Runner removal


√ Runner removed successfully
√ Removed .credentials
√ Removed .runner
....
# Authentication
√ Connected to GitHub
# Runner Registration
√ Runner successfully added
√ Runner connection is good
# Runner settings
√ Settings Saved.
√ Connected to GitHub

Current runner version: '2.284.0'
2021-11-17 20:11:30Z: Listening for Jobs
```

Reply:
I can not be sure on a super-long term, I haven't looked into the token and don't know it has expiration time. Though on 6d of run killing pid produces a successful restart as you can see above. Token is updated by the operator, should be... So  worked fine for me.

Reply:
Hm, I thought `--replace` should do the same trick, but in one command.